NFS integration tests failing in master

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The NFS integration tests are failing consistently in master since *.

```
--- FAIL: TestNfsSuite (863.80s)
    --- FAIL: TestNfsSuite/TestNfsServerInstallation (814.52s)
        read_write.go:45: 
            	Error Trace:	read_write.go:45
            	            				nfs_test.go:114
            	Error:      	Should be true
            	Test:       	TestNfsSuite/TestNfsServerInstallation
            	Messages:   	Make sure there are two read-write-test pods present in Running state
        require.go:794: 
            	Error Trace:	nfs_test.go:145
            	            				nfs_test.go:116
            	Error:      	Received unexpected error:
            	            	unable to write data to pod -- : Failed to run: kubectl [exec read-write-test-5755fb789d-9z9q2 -- cat /mnt/data] : Failed to complete 'kubectl': exit status 1. 
            	Test:       	TestNfsSuite/TestNfsServerInstallation
```

**Expected behavior:**
Integration tests should pass

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->
Look at the CI logs starting with master build [#1577](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/master/1577/pipeline/56)

This error is related:
```
2019-12-05 03:51:18.195066 I | exec: Running command: kubectl get pod -n default -l app=read-write-test -o jsonpath={.items[*].metadata.name}
2019-12-05 03:51:18.264858 I | exec: Running command: kubectl exec read-write-test-5755fb789d-9z9q2 -- cat /mnt/data
2019-12-05 03:51:18.364925 E | utils: Failed to execute: kubectl [exec read-write-test-5755fb789d-9z9q2 -- cat /mnt/data] : Failed to complete 'kubectl': exit status 1. . error: unable to upgrade connection: container not found ("alpine")
2019-12-05 03:51:18.364956 I | integrationTest: nfs volume read exited, err: unable to write data to pod -- : Failed to run: kubectl [exec read-write-test-5755fb789d-9z9q2 -- cat /mnt/data] : Failed to complete 'kubectl': exit status 1. . result: 
2019-12-05 03:51:18.364963 W | integrationTest: nfs volume read failed, will try again
```

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary
* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

 To get logs, use `kubectl -n <namespace> logs <pod name>`
When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
Read [Github documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod):
* Storage backend version (e.g. for ceph do `ceph -v`):
* Kubernetes version (use `kubectl version`):
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/master/ceph-toolbox.html)):

--- a/pkg/operator/nfs/server.go
+++ b/pkg/operator/nfs/server.go
@@ -25,7 +25,6 @@ import (
 
 const (
 	ganeshaLog     = "/dev/stdout"
-	ganeshaPid     = "/var/run/ganesha/ganesha.pid"
 	ganeshaOptions = "NIV_DEBUG"
 )
 
@@ -67,7 +66,7 @@ func Setup(ganeshaConfig string) error {
 func Run(ganeshaConfig string) error {
 	// Start ganesha.nfsd
 	logger.Infof("Running NFS server!")
-	cmd := exec.Command("ganesha.nfsd", "-p", ganeshaPid, "-F", "-L", ganeshaLog, "-f", ganeshaConfig, "-N", ganeshaOptions)
+	cmd := exec.Command("ganesha.nfsd", "-F", "-L", ganeshaLog, "-f", ganeshaConfig, "-N", ganeshaOptions)
 	if out, err := cmd.CombinedOutput(); err != nil {
 		return fmt.Errorf("ganesha.nfsd failed with error: %v, output: %s", err, out)
 	}

